docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,151 @@
+
+## Create a console application
+
+1. Create a **C# .NET Core console application** called "RestaurantViolations".
+
+## Prepare and understand the data
+
+> The data set used to train and evaluate the machine learning model is originally from the [San Francisco Department of Public Health Restaurant Safety Scores](https://www.sfdph.org/dph/EH/Food/score/default.asp). For convenience, the dataset has been condensed to only include the columns relevant to train the model and make predictions. Visit the following website to learn more about the [dataset](https://data.sfgov.org/Health-and-Social-Services/Restaurant-Scores-LIVES-Standard/pyih-qa8i?row_index=0).
+
+[Download the Restaurant Safety Scores dataset](https://github.com/luisquintanilla/machinelearning-samples/raw/AB1608219/samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantScores.zip) and unzip it.
+
+Each row in the dataset contains information regarding violations observed during an inspection from the Health Department and a risk assessment of the threat those violations present to public health and safety.
+
+| InspectionType | ViolationDescription | RiskCategory |
+| --- | --- | --- |
+| Routine - Unscheduled | Inadequately cleaned or sanitized food contact surfaces | Moderate Risk |
+| New Ownership | High risk vermin infestation | High Risk |
+| Routine - Unscheduled | Wiping cloths not clean or properly stored or inadequate sanitizer | Low Risk |
+
+- **InspectionType**: the type of inspection. This can either be a first-time inspection for a new establishment, a routine inspection, a complaint inspection, and many other types.
+- **ViolationDescription**: a description of the violation found during inspection. 
+- **RiskCategory**: the risk severity a violation poses to public health and safety.
+
+The `label` is the column you want to predict. When performing a classification task, the goal is to assign a category (text or numerical). In this classification scenario, the severity of the violation is assigned the value of low, moderate, or high risk. Therefore, the **RiskCategory** is the label. The `features` are the inputs you give the model to predict the `label`. In this case, the **InspectionType** and **ViolationDescription** are used as features or inputs to predict the **RiskCategory**.
+
+## Choose a scenario
+
+![Model Builder wizard in Visual Studio](./media/sentiment-analysis-model-builder/model-builder-screen.png)
+
+To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is `Issue Classification`.
```

Comment:
```suggestion
To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is *Issue Classification*.
```

Reply:
I don't believe the code formatting is needed, but italics for emphasis may work.